src/hooks/usePolledBalance.ts

```diff
@@ -379,3 +379,24 @@ export function useAntStakingMinimum(readOnly?: boolean): BigNumber | null {
 
   return minStakeAmount
 }
+
+export function useOptionConversionRate(): number {
+  const [rate, setRate] = useState(0.05)
+
+  const getBalance = useCallback(async (clear) => {
+    try {
+      const response = await fetch(
+        'https://datafeed-rinkeby.aragon.org/organizations'
+      )
```

Comment:
This URL should be a configuration so that it will work for mainnet, rinkeby or any supported network.  see https://github.com/aragon/upgrade/blob/master/src/environment/networks.ts for sample configuration

Reply:
I don't think this page is supposed to support multichain stuff. Once deployed, it will only be available for mainnet, with mainnet setting, etc.